llvm/lib/Transforms/Scalar/LoopDistribute.cpp

```diff
@@ -686,6 +686,9 @@ class LoopDistributeForLoop {
     if (!Dependences || Dependences->empty())
       return fail("NoUnsafeDeps", "no unsafe dependences to isolate");
 
+    LLVM_DEBUG(dbgs() << "LDist: Found a loop: " << L->getHeader()->getName()
```

Comment:
make clear that we found a loop LDIst can handle?

Reply:
Same here.